debian/copyright

```diff
@@ -0,0 +1,19 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: p4lang-pi
```

Comment:
I am guessing this should be p4lang-p4c?

Reply:
Fixed.